### PR TITLE
Issue #10478: Update inputs for SeparatorWrapCheckTest

### LINKED ISSUE #10478: Update inputs for SeparatorWrapCheckTest
Child of #10080

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #10478

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestDot.java [hunk 1]
```diff
@@ -27,7 +35,7 @@ public static void foo(int i, String s) throws FooExceptionForTestDot, BarExcept
 
     public void goodCaseForTestDot(int... aFoo)throws FooExceptionForTestDot, BarExceptionForTestDot
     {
-        String s = "ffffooooString";
+        String s = "ffffooooString"; // violation
```
Comment (shashwatj07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestDot.java:38]: Violation is on line 38. Can't place comment.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestMethodRef.java [hunk 2]
```diff
@@ -13,7 +21,7 @@ void goodCase() {
 
     void badCase() {
         String[] stringArray = { "Barbara", "James", "Mary", "John",
-            "Patricia", "Robert", "Michael", "Linda" };
+            "Patricia", "Robert", "Michael", "Linda" }; // violation
```
Comment (shashwatj07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestMethodRef.java:24]: same

**Review by shashwatj07 — COMMENTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestDot.java (hunk 1 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapForTestDot.java:38]: @shashwatj07 , please keep reference to all unfortunate cases at https://github.com/checkstyle/checkstyle/issues/10378

**Pull request merged**